src/dbos-runtime/cli.ts

```diff
@@ -224,6 +225,17 @@ workflowCommands
     console.log(JSON.stringify(output));
   });
 
+workflowCommands
+  .command('steps')
+  .description('List the steps of a workflow')
+  .argument('<uuid>', 'Target workflow ID')
```

Comment:
We're trying to remove the concept of UUIDs for IDs

```suggestion
  .argument('<id>', 'Target workflow ID')
```

Reply:
To do it in this PR would make it inconsistent with other commands.
I suggest we do it another PR and update all commands